traitsui/wx/scrubber_editor.py

```diff
@@ -461,13 +459,12 @@ def _enter_window(self, event):
         self._hover = True
 
         self.control.SetCursor(wx.Cursor(wx.CURSOR_HAND))
-
+        
         if not self._ignore_focus:
             self._ignore_focus = True
             self.control.SetFocus()
-
         self._ignore_focus = False
-
+        
```

Comment:
Not a request change, just a comment for future consideration:
Flake8 will complain about these trailing whitespaces (yep we should run flake8 in CI, but we don't at the moment). I think my IDE actually automatically removes trailing whitespaces for me when I save a file, might be something that is useful to set up once and for all.